### ISSUE TITLE
No support in postgres for alter default queries

### ISSUE THREAD

**Issue by Cabeda** [sqlfluff/sqlfluff#1074]
-- If this is a parsing or linting issue, please include a minimal SQL example which reproduces the issue, along with the `sqlfluff parse` output, `sqlfluff lint` output and `sqlfluff fix` output when relevant. Finally, please add any labels that are related to the issue before submitting so that it can be addressed easily! --

## Expected Behaviour
Shouldn't throw `Found unparsable section`.

## Observed Behaviour
Query throws `Found unparsable section`

## Steps to Reproduce

1. Create SQL file with the query `ALTER DEFAULT PRIVILEGES IN SCHEMA dwh GRANT SELECT ON TABLES TO "ro-{{params.env}}-role";`
2. Lint the SQL file with sqlfluff

## Version
sqlfluff 0.5.2
python 3.9.5

## Configuration
```
[sqlfluff]
dialect = postgres

[sqlfluff:templater:jinja:context]
params=temp
yesterday_ds=2021-02-08
```

**Comment by creste** [comment]
This was fixed by https://github.com/sqlfluff/sqlfluff/pull/1350.  I confirmed this SQL is now parsed correctly:
```sql
ALTER DEFAULT PRIVILEGES IN SCHEMA dwh GRANT SELECT ON TABLES TO "ro-{{params.env}}-role";
```

See https://github.com/sqlfluff/sqlfluff/pull/1350/files#diff-2d6b7ff056126cbb54e2dad5ff2c70f6d19095790b8eca845ef4727fd7f4d143 for the full test suite showing other variants of `ALTER DEFAULT PRIVILEGES` that are supported.
